pkgs/applications/graphics/inkscape/extensions/silhouette/default.nix

```diff
@@ -0,0 +1,78 @@
+{ fetchFromGitHub
+, lib
+, pkgs
+, python3
+, umockdev
+, writeScript
+}:
+
+let
+  launch-sendto_silhouette = writeScript "sendto_silhouette.sh" ''
+    cd $(dirname $0)
+    ./sendto_silhouette.py "$@"
+  '';
+  launch-silhouette_multi = writeScript "silhouette_multi.sh" ''
+    cd $(dirname $0)
+    ./silhouette_multi.py "$@"
+  '';
+in
+python3.pkgs.buildPythonApplication rec {
+  pname = "inkscape-silhouette";
+  version = "1.28.pre+unstable=2023-06-03"; # Note: there has been lot of as-yet-not-released development since 1.27 was released: https://github.com/fablabnbg/inkscape-silhouette/releases/tag/v1.27
```

Comment:
This may not be my personal favorite way to denote snapshot versions, but as there is [no consensus](https://github.com/NixOS/nixpkgs/issues/68518) on the 'correct way' IMHO this is as good as any :+1:

Reply:
Ooh, I'm not sure where I got this from, but I was following some other package in nixpkgs.

[According to @jtojnar](https://github.com/jfly/nixpkgs/blob/0ab50cfea23ff108aee2888125f5b9187af8a220/doc/languages-frameworks/python.section.md?plain=1#L726-L727), there *is* a recommendation, so I'll update this to follow that!